DQM/SiStripMonitorDigi/src/SiStripMonitorDigi.cc

```diff
@@ -291,23 +294,23 @@ void SiStripMonitorDigi::endLuminosityBlock(const edm::LuminosityBlock& lb, cons
     int nFeds [6] = { 96 , 96 , 84 , 15 , 15 , 134 }; // tec- , tec+ , tib , tid- , tid+ , tob
     int nFedsConnected [6] = { nFedTECm , nFedTECp , nFedTIB , nFedTIDm , nFedTIDp , nFedTOB };
 
-    MonitorElement * me = dqmStore_->get(topFolderName_+"/MechanicalView/NumberOfDigisInLastLS");     
```

Comment:
Yes. It isn't your change which introduced this, but you are touching the code and it makes you somewhat responsible for it :)
The sooner it is fixed, the better.